cpp/dependency_scan/metadata/README.md

```
@@ -0,0 +1,758 @@
+Unknown artifact in `scan_output/unknown.yml`:
+```yaml
+artifacts:
+  - tensorrt_llm/libs/executor_worker.cpython-312-x86_64-linux-gnu.so
+```
+
+Create `tensorrt-llm.yml`:
+```yaml
+name: tensorrt-llm
+version: "1.2.0"
+description: TensorRT-LLM inference optimization libraries
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)